dev/com.ibm.ws.crypto.ltpakeyutil/src/com/ibm/ws/crypto/ltpakeyutil/LTPACrypto.java

```diff
@@ -594,9 +594,9 @@ protected static final void setRSAKey(byte[][] key) {
     private static SecretKey constructSecretKey(byte[] key, String cipher)
             throws InvalidKeyException, NoSuchAlgorithmException, InvalidKeySpecException, NoSuchProviderException {
         SecretKey sKey = null;
-        if (cipher.indexOf("AES") != -1) {
-            // 16 bytes = 128 bit key
-            sKey = new SecretKeySpec(key, 0, 16, "AES");
+        if (cipher.indexOf(CryptoUtils.ENCRYPT_ALGORITHM_AES) != -1) {
+            int keyLength = fipsEnabled ? CryptoUtils.AES_256_KEY_LENGTH_BYTES : CryptoUtils.AES_128_KEY_LENGTH_BYTES;
```

Comment:
`fipsEnabled` should probably be renamed to be more specific to 140-3. 

Fips 140-2 is supported but falls under the old key lengths, algorithms, etc.

Reply:
we'll revisit the naming at a later time.